lib/inferno/ext/fhir_client.rb

```diff
@@ -3,11 +3,11 @@ class Client
     attr_accessor :oauth_credentials, :auth_info
 
     def need_to_refresh?
-      oauth_credentials&.need_to_refresh?
+      auth_info&.need_to_refresh?.present? || oauth_credentials&.need_to_refresh?.present?
```

Comment:
These shouldn't need `.present?` since `need_to_refresh?` should return `true/false`.

Reply:
yeah, just wanted  a consistent `true/false` output for unit testing instead of occasionally getting `nil` . I'll update and use other rspec matcher to check for falsey.

Reply:
An alternative would to wrap the whole thing in `!!`.